.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -21,7 +21,7 @@ If you are still having issues, please include as much information as possible:
    General, Core, and Other are also allowed as types
 2. OS type and version:
 3. Java version:
-4. spanner-jdbc version(s):
+4.  version(s):
```

Comment:
@thiagotnunes this seems to be a harmless bot update - but there is an extra whitespace. I'm lgtm'ing the PR. Tet me know if there is more to be done.